lib/tizen_doctor.dart

```diff
@@ -106,11 +110,11 @@ class TizenWorkflow extends Workflow {
   bool get appliesToHostPlatform => globals.platform.isLinux;
 
   @override
-  bool get canLaunchDevices => getSdbPath() != null;
+  bool get canLaunchDevices => tizenSdk != null;
 
   @override
-  bool get canListDevices => getSdbPath() != null;
+  bool get canListDevices => tizenSdk != null;
 
   @override
-  bool get canListEmulators => getSdbPath() != null;
+  bool get canListEmulators => tizenSdk != null && tizenSdk.emCli.existsSync();
```

Comment:
```emCli``` seems to need null checking.

Reply:
`emCli` is a null-safe property so this is okay.

Reply:
Thank you for checking. :+1: